test/topology/suite.yaml

```diff
@@ -1,5 +1,6 @@
 type: Topology
 pool_size: 4
+ci_pool_size: 10
```

Comment:
What is this number "10" based on? Did you try various numbers until you got the fastest run? Or what?
What if in the future we change our CI VMs?

Reply:
[Here](https://docs.google.com/document/d/1bHGbtshAwk81M3PyhGX5bR2oXQN9nGz8g-eGBPbnGXI/edit#heading=h.lme68hkepbv7) a very brief example of the several runs with different parameters.
If we change the VM's configuration, we will need to manually adjust the pool size.